classes/simplemap.php

```diff
@@ -231,29 +231,29 @@ function location_search_form( $atts ) {
 
 			// Normal Field inputs.
 			$ffi['street']   = array(
-				'label' => apply_filters( 'sm-search-label-street', __( 'Street: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-street', __( 'Street: ', 'SimpleMap' ), $post ),
 				'input' => '<input type="text" id="location_search_address_field" name="location_search_address" value="' . esc_attr( $address_value ) . '" />',
 			);
 			$ffi['city']     = array(
-				'label' => apply_filters( 'sm-search-label-city', __( 'City: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-city', __( 'City: ', 'SimpleMap' ), $post ),
 				'input' => '<input type="text"  id="location_search_city_field" name="location_search_city" value="' . esc_attr( $city_value ) . '" />',
 			);
 			$ffi['state']    = array(
-				'label' => apply_filters( 'sm-search-label-state', __( 'State: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-state', __( 'State: ', 'SimpleMap' ), $post ),
 				'input' => '<input type="text" id="location_search_state_field" name="location_search_state" value="' . esc_attr( $state_value ) . '" />',
 			);
 			$ffi['zip']      = array(
-				'label' => apply_filters( 'sm-search-label-zip', __( 'Zip: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-zip', __( 'Zip: ', 'SimpleMap' ), $post ),
 				'input' => '<input type="text" id="location_search_zip_field" name="location_search_zip" value="' . esc_attr( $zip_value ) . '" />',
 			);
 			$ffi['country']  = array(
-				'label' => apply_filters( 'sm-search-label-country', __( 'Country: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-country', __( 'Country: ', 'SimpleMap' ), $post ),
 				'input' => '<input type="text" id="location_search_country_field" name="location_search_country" value="' . esc_attr( $country_value ) . '" />',
 			);
 			$ffi['empty']    = array( 'label' => '', 'input' => '', );
 			$ffi['submit']   = array(
 				'label' => '',
-				'input' => '<input type="submit" value="' . apply_filters( 'sm-search-label-search', __( 'Search', 'simplemap' ), $post ) . '" id="location_search_submit_field" class="submit" />',
+				'input' => '<input type="submit" value="' . apply_filters( 'sm-search-label-search', __( 'Search', 'SimpleMap' ), $post ) . '" id="location_search_submit_field" class="submit" />',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 196 characters

<!-- Comment by [SideCI](https://sideci.com) -->